app/models/store.rb

```diff
@@ -0,0 +1,43 @@
+class Store
+  def initialize(session)
+    @session = session
+  end
+
+  def candidate_info
+    x = @session[:registration]
```

Comment:
Is it worth renaming the variables in the individual steps to be consistent with the API? Then you could drop this big mapping function?

Reply:
It will also save you mapping the other way around when you come to pre-fill the form with the existing candidate data from the API

Reply:
Yes, if We'd known the names and types could have done it from the get go, plan is to get it working then refactor